packages/xod-client/src/project/messages.js

```diff
@@ -0,0 +1,3 @@
+export const PROJECT_HAS_INVALID_NODETYPES_IN_SOME_PATCHES = 'Project has invalid NodeTypes in some Patches';
```

Comment:
I would be confused if see the message. Much better if it would be: “Patch @/foo can’t resolve its reference to bu/ba/ga”. Can we improve it such way?